scripts/pre_commit_linter.py

```diff
@@ -111,10 +111,10 @@
     }
 ]
 
-BAD_PATTERNS_APP_FECONF = {
-    'FORCE_DEV_MODE = False': {
-        'message': 'Please set the FORCE_DEV_MODE variable in feconf.py '
-                   'to true before committing.',
+BAD_PATTERNS_FECONF = {
+    'FORCE_DEV_MODE = True': {
```

Comment:
Please check that this actually triggers when the variable is changed.

Reply:
my bad. Fixed